### ISSUE TITLE
Enhanced documentation

### ISSUE THREAD

**Issue by Jonghakseo** [Jonghakseo/chrome-extension-boilerplate-react-vite#645]
**Is your feature request related to a problem? Please describe.**

As reported in several issues, creating the extension itself, and using this boilerplate in the process, requires more explanation.

**Describe the solution you'd like**

More detailed and improved documentation is needed. We are thinking of documentation based on each package's README.md, but it may be necessary to provide a self-hosted documentation page.

**Describe alternatives you've considered**

I think we can share information in the Discord community before documenting it.

**Additional context**

**Comment by PatrykKuniczak** [comment]
Basing on that comment: https://github.com/Jonghakseo/chrome-extension-boilerplate-react-vite/issues/651#issuecomment-2314495499

I think good idea is to add tutorials to the most popular lib, as MUI/Chakra UI/Radix and so on to new DOCS as we have on the `legacy` version

**Comment by Jonghakseo** [comment]
@PatrykKuniczak 

I agree. I want to turn Markdown files in this repository into a webpage as it is. 

https://github.com/wranders/markdown-to-pages-action

I'm looking at something like the project above.

**Comment by PatrykKuniczak** [comment]
> @PatrykKuniczak 
> 
> I agree. I want to turn Markdown files in this repository into a webpage as it is. 
> 
> https://github.com/wranders/markdown-to-pages-action
> 
> I'm looking at something like the project above.

I haven't heard about it in the past, that's interesting, because we don't need any new hosting, github pages are free 😁

**Comment by dandv** [comment]
I've [improved the README a bit](https://github.com/Jonghakseo/chrome-extension-boilerplate-react-vite/pull/791) but I agree that the documentation assumes users know how Chrome extensions work. Maybe it's better that way, to avoid everyone and their dog from publishing crappy half-baked extension.
